recipes/dpt/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set name = "DPT" %}
+{% set version = "1_0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/isl-org/DPT/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 32cceaae9c1717be674008b0644382b93b455d98c3f24eea547464082bd0bb16
+  - url: https://raw.githubusercontent.com/isl-org/DPT/main/setup.py
+    sha256: 2364ff07e09b422d1896038e684797707782fa427b8396e329d94155b96320db
+    path: setup.py
+  - url: https://github.com/isl-org/DPT/releases/download/1_0/dpt_hybrid-ade20k-53898607.pt
+    sha256: 53898607bfff539504639b34eb1cd3d13fdb97f16c044462f017ed79942c8b94
+    folder: weights
+  - url: https://github.com/isl-org/DPT/releases/download/1_0/dpt_hybrid-midas-501f0c75.pt
+    sha256: 501f0c75b3bca7daec6b3682c5054c09b366765aef6fa3a09d03a5cb4b230853
+    folder: weights
+  - url: https://github.com/isl-org/DPT/releases/download/1_0/dpt_hybrid_kitti-cb926ef4.pt
+    sha256: cb926ef4393a23358e6dd9ad579a1e26339771aea12b8c9bc4884150fd16be07
+    folder: weights
+  - url: https://github.com/isl-org/DPT/releases/download/1_0/dpt_hybrid_nyu-2ce69ec7.pt
+    sha256: 2ce69ec7eaa608a9aad6549655c39a40537e0040ba602a450bcd0e26390d3384
+    folder: weights
+  - url: https://github.com/isl-org/DPT/releases/download/1_0/dpt_large-ade20k-b12dca68.pt
+    sha256: b12dca688686e8af6758a32b437368c0a6ebf5fde9f8ab7819e0dd9dab8eda22
+    folder: weights
+  - url: https://github.com/isl-org/DPT/releases/download/1_0/dpt_large-midas-2f21e586.pt
+    sha256: 2f21e586477d90cb9624c7eef5df7891edca49a1c4795ee2cb631fd4daa6ca69
+    folder: weights
+  
+build:
+  number: 0
+  skip: true  # [win]
```

This looks like it's pure Python and could be noarch (we can ignore the Windows CI failure due to missing pytorch):
```suggestion
  noarch: python
```

---

Awesome changes made, I didn't know if the windows failure could be ignore, so I skipped win. Thanks for the review!

---

Noarch packages are only (currently at least) built on Linux anyways, so Windows users trying to install it just get a missing dependency error. In exchange, the build resources needed for this package (and the amount of rebuilds) is much much less.